pom.xml

```diff
@@ -21,6 +21,7 @@
         <module>webconsole</module>
         <module>jsonlogging</module>
         <module>prometheus-webconsole</module>
+        <module>consul</module>
```

Comment:
Added explicit dependency in consul and logging pom, version managed by the root pom.